module/metrics/transaction_validation.go

```diff
@@ -0,0 +1,45 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module"
+)
+
+// TransactionValidationCollector TODO
+type TransactionValidationCollector struct {
+	transactionValidated        *prometheus.CounterVec
+	transactionValidationFailed *prometheus.CounterVec
+}
+
+// interface check
+var _ module.TransactionValidationMetrics = (*TransactionValidationCollector)(nil)
+
+func NewTransactionValidationCollector() *TransactionValidationCollector {
+	return &TransactionValidationCollector{
+		transactionValidated: promauto.NewCounterVec(prometheus.CounterOpts{
+			Name:      "transaction_validated_total",
```

Comment:
the metric is tracking successful validations. I think using `total` here will give people the impression that it's the total validation attempts.

Can you rename this to something like `transaction_validation_succeeded`

Reply:
The `total` suffix is part of Prometheus [best practices](https://prometheus.io/docs/practices/naming/#metric-names) for counters (guessing that's why it was there to begin with). 

Something like `transaction_validation_successes_total` would communicate both that it's a counter and that it's counting successes (not validations in general, like Peter pointed out).